packages/blocks/src/api/factory.js

```diff
@@ -212,6 +232,51 @@ const getBlockTypesForPossibleToTransforms = ( blocks ) => {
 	return blockNames.map( ( name ) => getBlockType( name ) );
 };
 
+/**
+ * Determines whether transform is a "block" type
+ * and if so whether it is a "wildcard" transform
+ * ie: targets "any" block type
+ *
+ * @param  {Object} t Block transform object
+ * @return {boolean}   whether transform is a wildcard transform
+ */
+export const isWildcardBlockTransform = ( t ) => t && t.type === 'block' && t.blocks.length && t.blocks.includes( '*' );
+
+/**
+ * Determines whether the given Block is the core Block which
+ * acts as a container Block for other Blocks as part of the
+ * Grouping mechanocs
+ * @param  {string} name the name of the Block to test against
+ * @return {boolean}      whether or not the Block is the container Block type
+ */
+export const isContainerGroupBlock = ( name ) => name === 'core/group';
+
+/**
+ * Determines whether the provided Blocks are of the same type
+ * (eg: all `core/paragraph`).
+ *
+ * @param  {Array}  blocksArray the Block definitions
```

Comment:
Minor: This should be enforced better by our linting, but: There's no need to align the `@param` and `@return` statements, and in-fact they should be treated as distinct groups separated with a newline.

Reference: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

Reply:
Noted. Fixed. Thanks